classes/local/store/object_file_system.php

```diff
@@ -806,4 +806,33 @@ public function add_file_from_string($content) {
 
         return $result;
     }
+
+    /**
+     * Returns the total size of the filedir directory in bytes.
+     * @return float|int
+     */
+    public function get_filedir_size() {
+        $output = $this->exec_command('/usr/bin/du -sk ' . escapeshellarg($this->filedir) . ' | cut -f1');
```

Comment:
can you please use the $CFG->pathtodu cfg variable for this

Reply:
updated